presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -6427,6 +6427,14 @@ public void testCreateTableWithConstraints()
                 format("CONSTRAINT pk1 %s (c4) NOT ENFORCED", primaryKey),
                 format("%s (c3, c1) DISABLED NOT RELY", primaryKey));
         assertQueryFails(createTableWithTwoConstraintsSql, "Multiple primary key constraints are not allowed");
+
+        assertUpdate(getSession(), createTableWithOneConstraintSql);
+        actualResult = computeActual("SHOW CREATE TABLE " + tableName);
+        assertEquals(getOnlyElement(actualResult.getOnlyColumnAsSet()), expectedcreateTableWithOneConstraint);
+        // Since PRIMARY is a non-reserved keyword, it gets parsed and then fails at column resolution
+        assertQueryFails("SELECT PRIMARY FROM " + tableName, ".*cannot be resolved.*");
```

Comment:
@rschlussel - is this what you had in mind?

Reply:
This is fine. I just meant to ensure that primary will work as a column name in a regular query.